### PR TITLE
Button label jsx

### PR DESCRIPTION
**Pull request by kaada**
Button label supports jsx, e.g. doing
`label={<Message>request-money.form.generate</Message>}`

### CODE REVIEW

File: src/button/Button.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ Button.propTypes = {
   state: Types.oneOf(Object.values(State)),
   block: Types.bool,
   onClick: Types.func.isRequired,
-  label: Types.string.isRequired,
+  label: Types.node.isRequired,
```
Comment (dozortsev) [inline src/button/Button.js]: with this change you introducing some breaking changes

Comment (dozortsev) [reply src/button/Button.js]: I would actually allow `string` label as well, just use `Types.oneOfType([Types.string, Types.node])...`

Comment (ProstoSanja) [reply src/button/Button.js]: https://reactjs.org/docs/typechecking-with-proptypes.html

Node: anything that can be rendered: string, number, element or array.

Isn't it redundant to add node+string?

Comment (dozortsev) [reply src/button/Button.js]: sorry, you're right

**Pull request merged**